lib/tbot/service_workload_identity_aws_ra.go

```diff
@@ -91,7 +91,20 @@ func (s *WorkloadIdentityAWSRAService) generate(ctx context.Context) error {
 	if err != nil {
 		return trace.Wrap(err, "marshaling private key")
 	}
-	svid, err := x509svid.ParseRaw(res.GetX509Svid().Cert, pkcs8)
+	certWithChain := new(bytes.Buffer)
+	_, err = certWithChain.Write(res.GetX509Svid().GetCert())
+	if err != nil {
+		return trace.Wrap(err, "writing leaf cert to buffer")
+	}
+	// If external PKI is configured, we need to append the chain to the leaf
+	// certificate before calling x509svid.ParseRaw.
+	for _, cert := range res.GetX509Svid().GetChain() {
+		_, err := certWithChain.Write(cert)
+		if err != nil {
+			return trace.Wrap(err, "writing chain cert to buffer")
+		}
+	}
```

Comment:
`(*bytes.Buffer).Write` is infallible 😌 

```suggestion
	_, _ = certWithChain.Write(res.GetX509Svid().GetCert())
	// If external PKI is configured, we need to append the chain to the leaf
	// certificate before calling x509svid.ParseRaw.
	for _, cert := range res.GetX509Svid().GetChain() {
		_, _ = certWithChain.Write(cert)
	}
```

Reply:
[204b0c7](https://github.com/gravitational/teleport/pull/54884/commits/204b0c75b61781fb54896c62f9925bfc021d4c9b)